python_scripts/replace_template.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Bot to replace all uses of a given template by a new template."""
+from __future__ import absolute_import, unicode_literals
+import sys
+import pywikibot
+from pywikibot import pagegenerators
+from pywikibot.tools.formatter import color_format
+
+if sys.version_info[0] > 2:
+    from queue import Queue
+    long = int
+else:
+    from Queue import Queue
+
+changed_pages = 0
```

Comment:
Unless there's some clever logic that I'm missing, these aren't actually producing any output.

Reply:
Ha, I wondered when that disapeared. I'll tuck an output to the end of ´main()`